src/web/handlers/TapGestureHandler.ts

```diff
@@ -99,7 +102,10 @@ export default class TapGestureHandler extends GestureHandler {
     ) {
       this.activate();
     } else {
-      this.delayTimeout = setTimeout(() => this.fail(), this.maxDelayMs);
+      this.delayTimeout = setTimeout(
+        () => this.fail(),
+        this.maxDelayMs
+      ) as unknown as number;
```

Comment:
Same

Reply:
Done in [f6a46e5](https://github.com/software-mansion/react-native-gesture-handler/pull/2873/commits/f6a46e59bf7ebe489416d68fc2700e18d5bb3d05)